### PR TITLE
Example_dp2.x_lmp_gaussian

### PR DESCRIPTION
**Pull request by ziqi-hu**

### CODE REVIEW

**Review by wanghan-iapcm — CHANGES_REQUESTED** [review]
Could you please check the new examples by `test_check_examples.py`? We do not want to manually check them.

**Comment by HuangJiameng** [comment]
image

It seems that you bring the changes when you switch to this branch.

**Comment by HuangJiameng** [comment]
> First-time contributors need a maintainer to approve running workflows

First-time contributors need a maintainer to approve running workflows. I think he has checked the example with unit tests.

**Comment by njzjz** [comment]
Please fix the errors reported by the unit test. You can firstly test on your local machine before pushing commits.

File: dpgen/generator/arginfo.py [hunk 1]
```diff
@@ -41,6 +42,8 @@ def data_args() -> List[Argument]:
     doc_sys_batch_size = 'Each number is the batch_size for training of corresponding system in sys_configs. If set to auto, batch size will be 32 divided by number of atoms.'
 
     return [
+        Argument('init_multi_systems', bool, optional=True,
```
Comment (njzjz) [inline dpgen/generator/arginfo.py:45]: `init_multi_systems` has been deprecated in #730.

**Review by njzjz — CHANGES_REQUESTED** [review]

File: dpgen/generator/arginfo.py (hunk 1 above)
Comment (ziqi-hu) [reply dpgen/generator/arginfo.py:45]: init_multi_systems still work in dpgen run, why deleted.

File: doc/troubleshooting/common-errors.md [hunk 2]
```diff
@@ -28,4 +28,7 @@ Some common reasons are as follows:
 2. You may have something wrong in your input files, which causes the process to fail.
 
 ## RuntimeError: find too many unsuccessfully terminated jobs.
-The ratio of failed jobs is larger than ratio_failure. You can set a high value for ratio_failure or check if there is something wrong with your input files. 
\ No newline at end of file
+The ratio of failed jobs is larger than ratio_failure. You can set a high value for ratio_failure or check if there is something wrong with your input files. 
+
+## Dargs: xxx is not allowed in strict mode.
+Strict format check has been applied since version 0.10.6.  To avoid misleading users, some older-version keys that are already ignored or absorbed into default settings are not allowed to be present. And the expected structure of the dictionary in the param.json also differs from those before versions 0.10.6. This error will occur when format check finds older-fashion keys in the json file.  Please try deleting or annotating these keys, or correspondingly modulate the json file. Example files in the newest format could be found in [examples](https://github.com/deepmodeling/dpgen/tree/master/examples).
```
Comment (njzjz) [inline doc/troubleshooting/common-errors.md:34]: Please revert changes to this file as you have submitted them in #885.

**Review by wanghan-iapcm — APPROVED** [review]

**Review by njzjz — APPROVED** [review]

**Pull request merged**